.azure-pipelines/steps/install-dotnet-sdk-32bit.yml

```diff
@@ -0,0 +1,30 @@
+parameters:
+  - name: sdkVersion
+    type: string
+    default: ''
+
+  - name: channel
+    type: string
+    default: ''
```

Comment:
This won't work as expected if you pass both a version and a channel. It will run two installed, one with each argument, instead of a single install with both.

Reply:
Maybe I'm missing something and I understand the difference 😄

Reply:
Correct, you need to pick one 😛 

Specifying both doesn't makes sense, so I could add a check that you don't specify both and just error

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/1563/commits/5de37199d1a2aa77f560cc2de418e38af3b93dc1

Reply:
Thanks!